rotkehlchen/chain/evm/decoding/uniswap/utils.py

```diff
@@ -0,0 +1,104 @@
+import logging
+from collections.abc import Callable
+from typing import TYPE_CHECKING
+
+from rotkehlchen.chain.ethereum.utils import asset_normalized_value
+from rotkehlchen.chain.evm.decoding.structures import DEFAULT_DECODING_OUTPUT, DecodingOutput
+from rotkehlchen.chain.evm.decoding.uniswap.constants import CPT_UNISWAP_V2, CPT_UNISWAP_V3
+from rotkehlchen.chain.evm.decoding.utils import maybe_reshuffle_events
+from rotkehlchen.errors.asset import UnknownAsset, WrongAssetType
+from rotkehlchen.history.events.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+
+if TYPE_CHECKING:
+    from rotkehlchen.assets.asset import CryptoAsset
+    from rotkehlchen.history.events.structures.evm_event import EvmEvent
+
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+def decode_basic_uniswap_info(
+        amount_sent: int,
+        amount_received: int,
```

Comment:
you didn't change anything ehre right? just moving it?

Reply:
this is the only additional change
![image](https://github.com/rotki/rotki/assets/55619686/ef7be32c-9107-40d2-b297-d5c6d3d80350)